pennylane/transforms/adjoint.py

```diff
@@ -36,8 +36,17 @@ def my_ops():
             qml.RX(0.123, wires=0)
             qml.RY(0.456, wires=0)
 
-        adj_my_ops = qml.adjoint(my_ops)
-        adj_my_ops() # This will queue ops [qml.RY(-0.456), qml.RX(-0.123)]
+        with qml.tape.QuantumTape() as tape:
+            my_ops()
+
+        with qml.tape.QuantumTape() as tape_adj:
+            qml.adjoint(my_ops)()
+
+    >>> print(tape.operations)
+    [RX(0.123, wires=[0]), RY(0.456, wires=[0])]
+
+    >>> print(tape_adj.operatioins)
+    [RY(-0.456, wires=[0]), RX(-0.123, wires=[0])]

```

Comment:
```suggestion
    >>> print(tape.operations)
    [RX(0.123, wires=[0]), RY(0.456, wires=[0])]
    >>> print(tape_adj.operatioins)
    [RY(-0.456, wires=[0]), RX(-0.123, wires=[0])]
```

Might as well make this a single code block